tools/export/gccarm.py

```diff
@@ -121,6 +121,8 @@ class GccArm(Exporter):
 
     DOT_IN_RELATIVE_PATH = True
 
+    MBED_CONF_ACTIVE = True
```

Comment:
This should really be `MBED_CONF_HEADER_SUPPORTED`. If present, this field says "this exporter can use the header files generated by the configuration system". If not present, the exporter can still work with the configuration system, but using the old method (direct generation of macros for the configuration system).